crates/forge/tests/data/print_test/tests/test_print.cairo

```diff
@@ -1,80 +1,69 @@
-use array::ArrayTrait;
-use snforge_std::PrintTrait;
-
 #[test]
 fn test_print() {
```

Comment:
Do we need it? It was here to test the `PrintTrait`. Once it is gone we can delete it too

Reply:
We should remove it